extensions/oidc-db-token-state-manager/runtime/src/main/java/io/quarkus/oidc/db/token/state/manager/runtime/OidcDbTokenStateManagerRunTimeConfig.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.oidc.db.token.state.manager.runtime;
+
+import static io.quarkus.runtime.annotations.ConfigPhase.RUN_TIME;
+
+import java.time.Duration;
+
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+@ConfigMapping(prefix = "quarkus.oidc-db-token-state-manager")
+@ConfigRoot(phase = RUN_TIME)
+public interface OidcDbTokenStateManagerRunTimeConfig {
+
+    /**
+     * Whether Quarkus should delete expired token states.
+     * You can disable this feature when you want only one of your service instances to delete expired tokens.
+     */
+    @WithDefault("true")
+    boolean deleteExpired();
+
+    /**
+     * How often should Quarkus check for expired tokens. By default, the Quarkus will check every 20 seconds.
+     */
+    @WithDefault("20")
+    Duration deleteExpiredDelay();
```

Comment:
I think this is a very frequent scan period. This is how the expired token refresh works. Lets say the ID token lifespan is 1 hour, and the users want the expired ID token refreshed. If we set the cookie max age to 1 hour, the same as that of the ID token (a small leeway is added to the cookie max-age by default) then in 1 hour and 5 mins the cookie itself will be gone and no refresh will happen. 
To deal with such cases, users extend the cookie max age with the session age property lets say by 24 hours, so even if the ID token has expired - it will still reach Quarkus with the session cookie and now Quarkus can give it a try and refresh it when it sees ID token has expired. The other strategy is a pre-emptive auto-refresh - SPA in the background thread keeps pinging Quarkus every 10 mins for ex, the session cookie with ID token is sent along the way by the browser, Quarkus checks, will ID token expire within the configured period of time, say 30 mins, if yes, refresh it even if the token is still valid.

None of these strategies will work if the record with the expired ID token has gone.

I believe it should be set to say 8 hours and users can tune, we could've checked the session age extension property but this session age extension can vary across multiple tenants I guess, but users will know the average value and  set this property accordingly.

Reply:
Done.